src/applications/disability-benefits/new-686/config/form.js

```diff
@@ -0,0 +1,56 @@
+// import fullSchema from 'vets-json-schema/dist/21-686-schema.json';
+import IntroductionPage from '../containers/IntroductionPage';
+import ConfirmationPage from '../containers/ConfirmationPage';
+
+// const { } = fullSchema.properties;
```

Comment:
If this is unnecessary, should it be deleted instead of commented out?

Reply:
Both of these are used in a final form, this branch is just for us to test out building this form from scratch but we know if we end up keeping this code that we will need these two files.